packages/dxf-deserializer/instantiate.js

```diff
@@ -0,0 +1,584 @@
+/*
```

Comment:
so much work ! :)

Reply:
are all those versions official supported still ?

Reply:
It’s not so much supported but the range of applications creating DXF files is staggering. A robust implementation shouldn’t die, so I added simple tests.